src/scripts/main.js

```diff
@@ -1,3 +1,79 @@
 'use strict';
 
 // write code here
+
+const table = document.getElementsByTagName('table');
+
+const people = table[0].children[1].children;
+
+const headers = [...table[0].children[0].children[0].children];
```

Comment:
again, a lot of `children[0]` doesn't look clear
you may query all `th`, or query `thead` and get their children

also, it's better to save some values in variables
for example query `tbody`, save it in a variable and use it when you need any first level children